package.json

```diff
@@ -45,6 +45,7 @@
     "ejs": "^3.1.5",
     "express": "^4.16.3",
     "express-jwt": "^6.0.0",
+    "express-sanitizer": "^1.0.5",
```

Comment:
@jdauphant @alemangui  J'ai ajouté une librairie de sanitization. J'étais un peu réfractaire à l'installer ( vu qu'à la base c'était juste pour régler mon problème avec codeql) `HTML injection vulnerability due to user-provided value` mais je me dis que c'est utile et ça pourra être utile sur d'autre input de faire une sanitization.